pom.xml

```diff
@@ -424,6 +446,7 @@
                 <groupId>org.geotools</groupId>
```

Comment:
could you put a comment here that this is 'works with' and so has to stay provided scope?